eZ/Publish/Core/Repository/Repository.php

```diff
@@ -802,7 +802,6 @@ public function getSearchService()
         $this->searchService = new SearchService(
             $this,
             $this->searchHandler->contentSearchHandler(),
```

Comment:
Updated in https://github.com/ezsystems/ezpublish-kernel/commit/f98458b29a5b1f1eb85752586d3893f37b478f76